apache-geode/src/main/java/com/baeldung/geode/functions/PrimeNumber.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.geode.functions;
+
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.cache.execute.RegionFunctionContext;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class PrimeNumber implements Function {
```

Comment:
Is this file still necessary? If not, let's remove it.

Reply:
Ok Sure..